.github/workflows/Integration.yml

```diff
@@ -0,0 +1,62 @@
+name: Integration
+
+on:
+  push:
+    branches:
+      - main
+      - deployment
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  install-lint-build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

Comment:
You can now use the latest one v4.
So, plz update this to `actions/checkout@v4`